app/src/main/java/org/mozilla/tv/firefox/hint/HintBinder.kt

```diff
@@ -37,12 +38,29 @@ object HintBinder {
                                 .start()
                     }
                     .subscribe { shouldDisplay ->
-                        hintContainer.isVisible = true
                         val translationY = when (shouldDisplay) {
                             true -> 0f
                             false -> hintContainer.height.toFloat()
                         }
+
                         hintContainer.animate()
+                                // If the hint bar is appearing, set the hintContainer.isVisible
```

Comment:
nit: this comment is redundant to the code and likely to get outdated: I'd remove it.